test/unrunnableTestCases.csv

```diff
@@ -2103,7 +2103,7 @@ CastAs673b,Error: XPTY0004: Casting not supported from xs:string to xs:QName.
 CastAs677,Error: XPST0017: Function string-to-codepoints with arity of 1 not registered. No similar functions found.
 CastAs678,Error: XPST0017: Function string-to-codepoints with arity of 1 not registered. No similar functions found.
 CastFOCA0003-1,AssertionError: Expected executing the XPath "xs:integer(99e100)" to resolve to one of the expected results, but got Error: FOAR0002: can not cast 9.9e+101 to xs:integer, it is out of bounds for JavaScript numbers., AssertionError: expected [Function] to throw error including 'FOCA0003' but got 'FOAR0002: can not cast 9.9e+101 to xs:integer, it is out of bounds for JavaScript numbers.'.
-K-SeqExprCast-8,AssertionError: expected [Function] to throw error including 'XPST0081' but got 'Cannot read property \'na\' of undefined'
+K-SeqExprCast-8,AssertionError: expected [Function] to throw error including 'XPST0081' but got 'Cannot read property \'pa\' of undefined'
```

Comment:
The test is 
`   <test-case name="K-SeqExprCast-8">
      <description> A prefix is used which isn't declared.</description>
      <created by="Frans Englich" on="2007-11-26"/>
      <test>3 cast as prefixDoesNotExist:integer</test>
      <result>
         <error code="XPST0081"/>
      </result>
   </test-case>
`

I'll make an issue for it